packages/opentelemetry-core/src/trace/TracerDelegate.ts

```diff
@@ -29,20 +29,20 @@ export class TracerDelegate implements types.Tracer {
   // Wrap a tracer with a TracerDelegate. Provided tracer becomes the default
   // fallback tracer for when a global tracer has not been initialized
   constructor(
-    private readonly tracer: types.Tracer | null = null,
-    private readonly fallbackTracer: types.Tracer = new NoopTracer()
+    private readonly _tracer: types.Tracer | null = null,
```

Comment:
I believe I remember @rochdev asking that parameters not have _ in the name. Given that, should we separate the parameter and member values for these?

Reply:
Yes, this is usually the pattern I would recommend. This keeps the API clean while properly indicating that the field is internal and should not be accessed to JavaScript developers.

Reply:
So private params are w/o `_` prefix while private members should be start with `_` prefix, right?

Reply:
Yes, this means that you won't be able to use the TS shorthand for properties in the constructor, but need to explicitly assign the private variable to the passed in arg

Reply:
Correct. The shorthand is mostly useful in an application, where everything is in TypeScript. Unfortunately, for a library this assumption cannot be made.